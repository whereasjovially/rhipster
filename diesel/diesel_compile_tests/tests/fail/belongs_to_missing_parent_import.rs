#[macro_use]
extern crate diesel;

table! {
    foos {
        id -> Integer,
        bar_id -> Integer,
    }
}

#[derive(Associations)]
#[belongs_to(Bar)]
struct Foo {
    bar_id: i32,
}

fn main() {}
