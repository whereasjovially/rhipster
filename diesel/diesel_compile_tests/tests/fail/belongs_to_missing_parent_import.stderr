error[E0412]: cannot find type `Bar` in this scope
  --> $DIR/belongs_to_missing_parent_import.rs:12:14
   |
12 | #[belongs_to(Bar)]
   |              ^^^ not found in this scope
